src/tests/fut.rs

```diff
@@ -37,8 +37,8 @@ async fn test_rpm_builder_async() -> Result<(), Box<dyn std::error::Error>> {
         )
         .await?
         .pre_install_script("echo preinst")
-        .add_changelog_entry("me", "was awesome, eh?", 123123123)
-        .add_changelog_entry("you", "yeah, it was", 12312312)
+        .add_changelog_entry("me", "was awesome, eh?", std::time::SystemTime::now())
+        .add_changelog_entry("you", "yeah, it was", std::time::SystemTime::now())
```

Comment:
This is why I wanted to go ahead and revert those changes, so it doesn't continually cause rebase issues.

Reply:
Let's merge your test refactor first and get that out of the way

Reply:
I'll split the PR in half so we can merge the refactors while I'm still working on the rewrites.